### PR TITLE
Table of category theory terminology

### PR DESCRIPTION
**Pull request by siddharthist**
This will help new contributors orient themselves and prevent easy misunderstandings!

### CODE REVIEW

**Comment by benediktahrens** [comment]
Can you explain the "classical maths" column?

**Comment by siddharthist** [comment]
@benediktahrens I got those analogies from the 2nd and 3rd paragraphs of the introduction to Chapter 9 of [the HoTT book](https://homotopytypetheory.org/book/).

**Comment by benediktahrens** [comment]
On 04/03/18 22:26, Langston Barrett wrote:
> @benediktahrens <https://github.com/benediktahrens> I got those 
> analogies from the 2nd and 3rd paragraphs of the introduction to Chapter 
> 9 of the HoTT book <https://homotopytypetheory.org/book/>.
> 

What does "classical maths" mean? Does it mean "maths in set theory"?

Comparing notions defined in type theory with notions defined in set 
theory is done by defining a translation of type theory in set theory or 
the other way round.
So what, say, a precategory corresponds to in set-theoretical maths 
depends on the chosen translation, e.g., to sets, or to simplicial sets.

I don't think this column is well specified, and the overview - which is 
good - will be better without it.

**Comment by siddharthist** [comment]
@benediktahrens Yes, I mean in set-theoretic foundations. Ok, I'll remove it!

File: UniMath/CategoryTheory/README.md [hunk 1]
```diff
@@ -12,6 +12,19 @@ http://arxiv.org/abs/1401.0053.
...
 
+## Terminology
+
+The terminology in this package differs from that
+of [the HoTT book](https://homotopytypetheory.org/book/).
+The following table offers a comparison.
+
+| UniMath                      | HoTT Book       | Classical math | Ob C | Hom_C | Univalence |
+|------------------------------|-----------------|----------------|------|-------|------------|
+| Precategory                  | n/a             | (∞,1)-category | Type | Type  | No         |
```
Comment (DanGrayson) [inline UniMath/CategoryTheory/README.md]: A precategory doesn't seem analogous to an (∞,1)-category.

**Comment by siddharthist** [comment]
Fixed!

File: UniMath/CategoryTheory/README.md [hunk 2]
```diff
@@ -12,6 +12,19 @@ http://arxiv.org/abs/1401.0053.
...
+
+The terminology in this package differs from that
+of [the HoTT book](https://homotopytypetheory.org/book/).
+The following table offers a comparison.
+
+| UniMath                      | HoTT Book       | Ob C | Hom_C | Univalence |
+|------------------------------|-----------------|------|-------|------------|
+| Precategory                  | n/a             | Type | Type  | No         |
+| Category                     | Precategory     | Type | Set   | No         |
+| Univalent/saturated category | Category        | Type | Set   | Yes        |
```
Comment (DanGrayson) [inline UniMath/CategoryTheory/README.md]: The word "saturated" is mentioned in a comment, but it goes a bit too far to say that it is the UniMath terminology for that concept.  What about putting just "univalent category" here?

**Pull request merged**